doc_source/fargate-getting-started.md

```diff
@@ -65,13 +65,16 @@ aws eks describe-cluster --name <cluster_name> --query cluster.resourcesVpcConfi
 
 ## Create a Fargate pod execution role<a name="fargate-sg-pod-execution-role"></a>
 
-When your cluster creates pods on AWS Fargate, the pods need to make calls to AWS APIs on your behalf to do things like pull container images from Amazon ECR\. The Amazon EKS pod execution role provides the IAM permissions to do this\.
+When your cluster creates pods on AWS Fargate, the kubelet that is running on the Fargate infrastructure needs to make calls to AWS APIs on your behalf to do things like pull container images from Amazon ECR\. The Amazon EKS pod execution role provides the IAM permissions to do this\.
```

Comment:
nit: i would leave this generic instead of Kubelet since different data plane processes now assume this role (fluent-bit comes to mind)

Reply:
"EKS Fargate components running on the Fargate infrastructure need to make calls to AWS APIs on your behalf" or something similar?